### PR TITLE
[EAM-714] Add controller for E-Signatures

### PR DESCRIPTION
**Pull request by BoykoBorisov**

### CODE REVIEW

File: eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java [hunk 1]
```diff
@@ -44,7 +45,21 @@ public Response createFollowUpWorkOrders(Activity activity) {
...
+		} catch (Exception e) {
+			return serverError(e);
+		}
+	}
+
+	@PUT
+	@Path("/esign")
+	@Produces("application/json")
+	@Consumes("application/json")
+	public Response eSignWorkOrderActivityChecklist(WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature){
```
Comment (pedroamarocern) [inline eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java:57]: `(WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature){` => `(WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature) {`

File: eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java [hunk 2]
```diff
@@ -44,7 +45,21 @@ public Response createFollowUpWorkOrders(Activity activity) {
...
+		try {
+			return ok(inforClient.getChecklistService().eSignWorkOrderActivityChecklist(authenticationTools.getInforContext(), workOrderActivityCheckListSignature));
+		} catch (InforException e){
```
Comment (pedroamarocern) [inline eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java:60]: `catch (InforException e){` => `catch (InforException e) {`

**Review by pedroamarocern — COMMENTED** [review]
Looks good to me, only minor issues code style issues.

File: eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java (hunk 1 above)
Comment (BoykoBorisov) [reply eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java]: Fixed.

File: eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java (hunk 2 above)
Comment (BoykoBorisov) [reply eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/ChecklistController.java]: Fixed.

**Pull request merged**